Update introduction.tex (#384)
worldbank/d4di: chapters/handling-data.tex

\section{Protecting confidence in development research}

The empirical revolution in development research\cite{angrist2017economic}
has therefore led to increased public scrutiny of the reliability of research.\cite{rogers_2017}\index{transparency}\index{credibility}\index{reproducibility}
Three major components make up this scrutiny: \textbf{reproducibility}\cite{duvendack2017meant}, \textbf{transparency},\cite{christensen2018transparency} and \textbf{credibility}.\cite{ioannidis2017power}
Development researchers should take these concerns seriously.
Many development research projects are purpose-built to address specific questions,
and often use unique data or small samples.
As a result, it is often the case that the data 
researchers use for such studies has never been reviewed by anyone else,
so it is hard for others to verify that it was 
collected, handled, and analyzed appropriately.

Reproducible and transparent methods are key to maintaining credibility 
and avoiding serious errors. 

> This is particularly true for research that relies on new data sources, 

Reviewer: ```suggestion
This is particularly true for research that relies on original or novel data sources, 
```

from innovative big data sources to surveys. 
The field is slowly moving in the direction of requiring greater transparency. 
Major publishers and funders, most notably the American Economic Association,
have taken steps to require that code and data
are accurately reported, cited, and preserved as outputs in themselves.\sidenote{
	\url{https://www.aeaweb.org/journals/policies/data-code}}

\subsection{Research reproducibility}

Can another researcher reuse the same code on the same data 

> and get the exact same results as in your published paper?\sidenote{	\url{https://blogs.worldbank.org/impactevaluations/what-development-economists-talk-about-when-they-talk-about-reproducibility}}

Reviewer: ```suggestion
and get the exact same results as in your published paper?\sidenote{
  \url{https://blogs.worldbank.org/impactevaluations/what-development-economists-talk-about-when-they-talk-about-reproducibility}}
```

This is a standard known as \textbf{computational reproducibility}, 
and it is an increasingly common requirement for publication.\sidenote{
\url{https://www.nap.edu/resource/25303/R&R.pdf}})
It is best practice to verify computational reproducibility before submitting a paper before publication. 
This should be done by someone who is not on your research team, on a different computer, 
using exactly the package of code and data files you plan to submit with your paper. 

> Code that is well-organized into a master do-file, and written to be easily run by others, 

Reviewer 2: ```suggestion
Code that is well-organized into a master script, and written to be easily run by others, 
```

is a great way to have new questions asked and answered
based on the valuable work you have already done.\sidenote{
	\url{https://blogs.worldbank.org/opendata/making-analytics-reusable}}

> Making your research reproducible is also a public good: \sidenote{

Reviewer: ```suggestion
Making your research reproducible is also a public good.\sidenote{
```

	\url{https://dimewiki.worldbank.org/Reproducible_Research}}

> enables other researchers to re-use your code and processes

Reviewer: ```suggestion
It enables other researchers to re-use your code and processes
```

\section{Ensuring privacy and security in research data}

> Anytime you are working with raw data in a development research project,

Reviewer: ```suggestion
Anytime you are working with original data in a development research project,
```

with a set of governance standards known as ``The Common Rule''.\sidenote{
	\url{https://www.hhs.gov/ohrp/regulations-and-policy/regulations/common-rule/index.html}}
If you interact with European institutions or persons,

> you will also become familiar with the General Data Protection Regulation ``GDPR'',\sidenote{

Reviewer: ```suggestion
you will also become familiar with the General Data Protection Regulation (GDPR),\sidenote{
```

Most commonly this consists of a formal application for approval of a specific
protocol for consent, data collection, and data handling.\sidenote{
  \url{https://dimewiki.worldbank.org/IRB_Approval}}
Which IRB has sole authority over your project is not always apparent,
particularly if some institutions do not have their own.
It is customary to obtain an approval from a university IRB
where at least one PI is affiliated,
and if work is being done in an international setting,
approval is often also required

> from an appropriate local institution subject to the laws of the country.

Reviewer: ```suggestion
from an appropriate local institution subject to the laws of the country where data originates.
```

The easiest way to reduce the risk of leaking personal information is to use it as rarely as possible.
It is often very simple to conduct planning and analytical work
using a subset of the data that has been \textbf{de-identified}.
We encourage this approach, because it is easy.
However, when PII is absolutely necessary to a task,
such as implementing an intervention
or submitting survey data,

> you must actively protect that data in transmission and storage.

Reviewer: ```suggestion
you must actively protect that information in transmission and storage.
```
